docs/dataset/advanced-usage/dataset-formatting/computer-vision.md

```diff
@@ -0,0 +1,193 @@
+---
+icon: kolena/media-16
+---
+# :kolena-media-20: Computer Vision
+
+In this document we will review best practices when setting up Kolena datasets for computer vision
+problems.
+
+## Basics
+
+### Supported File Data Formats
+
+The Kolena SDK supports upload of data in the Pandas [`DataFrame`](https://pandas.pydata.org/docs/reference/frame.html) format.
+
+The Kolena web app supports the following file formats.
+
+| Format    | Description                                              |
+|-----------|----------------------------------------------------------|
+| `.csv`     | Comma-separated values file, ideal for tabular data.     |
+| `.parquet` | Apache Parquet format, efficient for columnar storage.   |
+| `.jsonl`   | JSON Lines format, suitable for handling nested data.    |
```

Comment:
This buries the lede that what we mean by "CV" is:

- Images: `.png`, `.jpg`, etc. (everything supported by browsers)
- Point Clouds: `.pcd` files
- Videos: `.mp4`, `.mkv`, etc. (as supported by browser)

I would consider updating to have the supported data types and supported annotation types at the top of this doc, so users don't have to scroll below the fold to learn what we mean by CV.